concourse/model/traits/notifications.py

```diff
@@ -32,16 +33,39 @@
 )
 
 
-class NotificationTriggeringPolicy(enum.Enum):
+class NotificationTriggeringPolicy(AttribSpecMixin, enum.Enum):
```

Comment:
as just discussed via slack: extending this enum seems like a clever idea. my only objection would be that - different to all other attrib-specs we so far have - the names / attributes here do not refer to actual sub-attributes, but to values of one common attribute. How does the rendered documentation page look like?

Reply:
Changed the heading into `Attribute Values` in case of enum